flying-saucer-core/src/main/java/org/xhtmlrenderer/css/parser/property/AbstractPropertyBuilder.java

```diff
@@ -99,7 +101,7 @@ protected void checkIdentOrLengthType(CSSName cssName, CSSPrimitiveValue value)
 
     protected void checkIdentOrNumberType(CSSName cssName, CSSPrimitiveValue value) {
         int type = value.getPrimitiveType();
-        if (type != CSSPrimitiveValue.CSS_IDENT && type != CSSPrimitiveValue.CSS_NUMBER) {
+        if (type != CSSPrimitiveValue.CSS_IDENT && type != CSS_NUMBER) {
```

Comment:
The IDE didn't use a static import for both constants?

Reply:
Small steps, my friend, small steps. :)
IDE suggests million improvements. I don't want to apply all of them at once. One-by-one.